docs/artifacts/tutorials/private-powershell-library.md

```diff
@@ -179,6 +222,12 @@ We now have a private repository within Azure Artifacts that we can push our Pow
     ```
     
     > **NOTE:** You will notice above that the Publish and Source location both reference Version 2 of NuGet. PowerShell does not support Version 3 of NuGet.
+    
+    Another example for Azure Artifacts, which defaults to Version 3 of NuGet (but you will use Version 2):
```

Comment:
Can we change this to say more about when it would be used? e.g. "If you're still using the older visualstudio.com URLs, use this command instead:"

Reply:
Thank you, nice observation! I changed it. You're right that my wording wasn't specific enough for the scope of the sample URLs provided